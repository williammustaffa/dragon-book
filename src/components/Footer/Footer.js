import React from "react";
import { Grid, Segment, Container, } from "semantic-ui-react";

function Footer() {
  const year = new Date().getFullYear();

  return (
    <Segment inverted vertical id="footer">
      <Container>
        <Grid>
          <Grid.Row>
            <Grid.Column textAlign="center">
              Dragons Book Â© {year}
            </Grid.Column>
          </Grid.Row>
          </Grid>
      </Container>
    </Segment>
  );
}

export default Footer;