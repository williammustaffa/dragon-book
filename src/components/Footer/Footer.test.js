import React from "react";
import Footer from "./Footer";
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

test("Test Footer component", () => {
  const { asFragment, getByText } = render(<Footer />);

  // Run tests
  const year = new Date().getFullYear();

  expect(getByText(`Dragon Book Â© ${year}`)).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});