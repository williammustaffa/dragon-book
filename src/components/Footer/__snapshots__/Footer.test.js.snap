// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component 1`] = `
<DocumentFragment>
  <div
    class="ui inverted vertical segment custom-footer"
  >
    <div
      class="ui container"
    >
      <div
        class="ui grid"
      >
        <div
          class="row"
        >
          <div
            class="center aligned column"
          >
            Dragon Book Â© 2020
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
